<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VR Video Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="Player_StackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="enter_button">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ENTER</string>
        </property>
       </widget>
       <widget class="QComboBox" name="options_comboBox">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>160</y>
          <width>224</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Видео с компа на очки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Видео с вебки на очки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>С камеры очков на комп (и на очки)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Загрузка файлов (с командой запуска)</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="streamingPlayer">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="5">
         <widget class="QPushButton" name="stop_streaming_button">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Stop streaming</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="stop">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QPushButton" name="start_streaming_button">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Start streaming</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="connect_to_ver_button">
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Start stream to VR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="start_continue_button">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Start/Continue</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QComboBox" name="comboBox_coding">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="currentIndex">
           <number>5</number>
          </property>
          <item>
           <property name="text">
            <string>Video - H.264 + MP3(MP4)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video - H.264 + MP3(TS)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video - H.265 + MP3(MP4)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video - MPEG-2 + MPGA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Audio - MP3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video for Android HD</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pause">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="checkBox__enable_coding">
          <property name="text">
           <string>Включить перекодирование</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="VlcWidgetVolumeSlider" name="volume" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="openLocal">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Open local file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openUrl">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Open URL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="1">
         <widget class="VlcWidgetSeek" name="seek" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="VlcWidgetVideo" name="videoSurface" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="pause_streaming_button">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Pause streaming</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="stop_stream_to_vr">
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Stop stream to VR</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="send_to_server">
          <property name="minimumSize">
           <size>
            <width>97</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Send to server</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVolumeSlider.h</header>
   <container>1</container>
   <slots>
    <slot>mute()</slot>
    <slot>volumeUp()</slot>
    <slot>volumeDown()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
